test/data/invoice-hotel.json

```diff
@@ -117,30 +120,30 @@
 						"code": "VAT",
 						"rates": [
 							{
-								"key": "excluded",
+								"key": "exempt",
+								"ext": {
+									"it-sdi-nature": "N1"
+								},
 								"base": "1.00",
 								"amount": "0.00"
 							},
 							{
 								"key": "intermediate",
-								"base": "113.64",
+								"base": "113.6364",
 								"percent": "10.0%",
-								"amount": "11.36"
+								"amount": "11.3636"
 							}
 						],
-						"amount": "11.36"
+						"amount": "11.3636"
 					}
 				],
-				"sum": "11.36"
+				"sum": "11.3636"
 			},
 			"tax": "11.36",
 			"total_with_tax": "126.00",
 			"payable": "126.00",
 			"advance": "29.00",
 			"due": "97.00"
 		}
-	},
-	"sigs": [
-		"eyJhbGciOiJFUzI1NiIsImtpZCI6ImZmNzU5MTVmLWY2ZjgtNDU4Ny04OTBhLWNjZDhiODZiZjhmNSJ9.eyJ1dWlkIjoiNjc5YTJmMjUtNzQ4My0xMWVjLTk3MjItN2VhMmNiNDM2ZmY2IiwiZGlnIjp7ImFsZyI6InNoYTI1NiIsInZhbCI6ImQzNmM3NGUxOWJmNWYwMDk3NzM4Y2JmNTk3MGZiMjA1NDc5MmU5NDRiMmM3MWMwOWFmMGJjY2I0MmQzZGI1MzYifX0.iivLe0zUAMqlRmN4AJYgUIDhGcyBzYDIBJyTUj0sHBK3axNJC0JDPVKznfPOzShVfO3ILsvbuXxzVo-1sButvQ"
-	]
```

Comment:
I removed the signatures from the test envelopes as they weren't necessary, and rebuilding the files is (marginally) easier without them.